scripts/ts_bindings_check.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+set -e
+
+# This check is only used for CI.
+
+CWD="$(cd -P -- "$(dirname -- "${BASH_SOURCE[0]}")" && pwd -P)"
+
+cd "$CWD/../"
+
+# Export the ts-rs bindings
+cargo test --workspace export_bindings
+
+# Make sure no rows are returned
+! grep -nr --include=\*.ts ' | null' ./crates/
```

Comment:
Now anytime we tried to add an optional field, we can make sure that it never outputs `thing: Thing | null`, but `thing?: Thing`

@Nutomic you might want to consider moving your error grep check to something like this. There's not much reason it needs to run in rust code when we can add a woodpecker test for it.

Reply:
It would be quite tricky to get a list of enum variants via grep.